app/src/test/java/com/parasoft/demoapp/util/FooterUtilTest.java

```diff
@@ -0,0 +1,51 @@
+package com.parasoft.demoapp.util;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+import android.app.Activity;
+import android.content.res.Resources;
+import android.text.Editable;
+import android.widget.TextView;
+
+import com.parasoft.demoapp.BuildConfig;
+import com.parasoft.demoapp.R;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FooterUtilTest {
+    @Mock
+    private Activity activity;
+    @Mock
+    private Resources resources;
+    @Mock
+    private TextView textView;
+    @Captor
+    private ArgumentCaptor<Editable> captor;
+
+    @Test
+    public void setFooterInfo() {
+        // Given
+        String app_title = "PARASOFT DEMO APP";
```

Comment:
Please use R.string.app_title instead of fixed string here.

Reply:
It can be any text beacuse activity have been already mocked here. I think what really confused you is that I used wrong arguments. So now I have changed it to use 'anyInt()' like below.
![image](https://user-images.githubusercontent.com/109124579/188821720-50772b21-bcbb-4ab7-bd28-6eee27b045a2.png)